MAINTAINERS.md

```diff
@@ -0,0 +1,57 @@
+# Maintainers
+
+The following is the list of current maintainers for all [FireFly code repositories](https://github.com/hyperledger-labs?q=firefly):
+
+| Name              | GitHub          | Email                        | LFID              |
```

Comment:
I think this list might be too long to ensure deep skill in code-owners across the repos, considering the plugin nature, and the way we work naturally is with specialist ownership.
Let me net out what I think the situation is, and Nicko you can see the best way to translate that into the docs.

- @nguyer - you have a special role as community admin, meaning you retain administrative privledges on the repos.
- Core - @peterbroadhurst @nguyer @awrichar 
- Samples - @awrichar @peterbroadhurst @shorsher 
- CLI - @nguyer @awrichar @peterbroadhurst 
- DX - @gabriel-indik @peterbroadhurst 
- Ethconnect - @peterbroadhurst @vdamle @jimthematrix 
- Fabconnect - @jimthematrix @peterbroadhurst @vdamle 
- Cordaconnect - @panghalamit @peterbroadhurst @jimthematrix 
- UI - @shorsher @peterbroadhurst @awrichar

I note that because the docs are in Core, this will mean @nickgaski cannot complete a code-owner review on changes to the docs (the one person from the original list below not mentioned above). We can see if that causes friction, and then the right thing to do would probably be to split out the docs to its own repo.

Reply:
... wonder if the Samples list is correct, or whether @nguyer you should swap in for me there (or have both)

Reply:
Thanks for the feedback Peter. I will split the lists out to be specific to each repo, as well as update the `CODEOWNERS` file in each repo to reflect this.

Reply:
RE: the docs topic you raised... I can make @nickgaski a `CODEOWNER` of just the `./docs` directory. That would give him the ability to approve PRs to that directory and not the rest of the repo.